tests/libtest/test_elasticsearch.py

```diff
@@ -113,14 +127,30 @@ def smallfile_run(self, es):
 
     def test_elasticsearch(self):
         """
-        This test only deploy the elasticsearch module, connect to it with and
-        without credentials and teardown the environment
+        This test do the following operations:
 
-        Args:
-            es (fixture) : fixture that deploy / teardown the elasticsearch
+            * deploy the elasticsearch module
+            * connect to it
+            * run a simple SmallFile benchmark (to verify usability)
+            * dump the results to a file
+            * push the results from the file to the Dev. ES.
+            * teardown the environment
 
         """
 
+        log.info("Test with 'Dummy' Storageclass")
+        try:
+            self.es = ElasticSearch(sc="dummy")
+        except Exception:
+            log.info("Raised as expected !")
+
+        log.info("Test with 'Real' Storageclass")
+        try:
+            self.es = ElasticSearch()
+        except Exception as ex:
+            log.error("Raise as expected !")
+            raise ex
```

Comment:
@Avilir Can we add specific exception here https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/exceptions.py which will be more specific. Vasu mentioned that this might cause failure with Python3.8

Based on his comment I updated the code as in this line https://github.com/red-hat-storage/ocs-ci/blob/master/tests/e2e/scale/test_scale_12_OCS_worker_nodes_and_6000_PVCs.py#L97

Reply:
i will update it.

Reply:
This comment applies to line no 154 & 168 in ocs_ci/ocs/elasticsearch.py file too.